pikaur/prompt.py

```diff
@@ -1,26 +1,79 @@
 import sys
+import tty
 from typing import Callable, List
 
+from .args import PikaurArgs
+from .config import PikaurConfig
+
 from .core import interactive_spawn
 from .i18n import _
 from .pprint import color_line, print_status_message
-from .args import PikaurArgs
 
 
-def ask_to_continue(text: str = None, default_yes=True, args: PikaurArgs = None) -> bool:
+def get_answer(question: str, answers: str='Yn') -> str:
```

Comment:
and also mb rename this to `get_answer_from_tty`/`read_answer_from_tty` or smth like `get_answer_interactively`

Reply:
I've also tried removing the type hint on `answers` to see if the issue with the build bot is that.

Reply:
Error went away, so that's the problem. Do you want me to stick it back in, or remove the others so the build officially passes?